.gitignore

```diff
@@ -18,7 +18,7 @@
 /build
 /dist
 /env
-*local.env
+surface/local.env
```

Comment:
`local.env` could be stored in other parts of the project (as users require it), so this change is sub-optimal because it will only ignore `local.env` files stored in `surface`.